config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml

```diff
@@ -9,6 +9,7 @@ periodics:
     - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180730-c8036c3a7
       args:
       - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/kubernetes-anywhere=kubeadm-e2e
```

Comment:
skimming through this PR I don't see us doing this for other k-a jobs (the presubmits?) we should find them and do this for all k-a jobs